scripts/setup.sh

```diff
@@ -1,290 +1,153 @@
+    as_user "${PIP} install -q --upgrade pip"
+    as_user "${PIP} install -q --upgrade \
+        ansible==${ANSIBLE_VERSION} \
+        Jinja2==${JINJA2_VERSION} \
+        netaddr \
+        ruamel.yaml \
+        PyMySQL \
+        selinux"
+else
+    echo "ERROR: Unable to create Python virtual environment, 'virtualenv' command not found"
+fi
+
+# Clone DeepOps git repo if running standalone
+if ! (cd "${SCRIPT_DIR}/.." && grep -i deepops README.md >/dev/null 2>&1 ) ; then
+    if command -v git &> /dev/null ; then
+        if ! test -d deepops ; then
+            as_user git clone --branch ${DEEPOPS_TAG} https://github.com/NVIDIA/deepops.git
         fi
+        cd deepops
+    else
+        echo "ERROR: Unable to check out DeepOps git repo, 'git' command not found"
+        exit
     fi
-    cd deepops
 fi
 
 # Install Ansible Galaxy roles
-ansible-galaxy --version >/dev/null 2>&1
-if [ $? -eq 0 ] ; then
+if command -v ansible-galaxy &> /dev/null ; then
     echo "Updating Ansible Galaxy roles..."
-    if [ $PROXY_USE -gt 0 ]; then
-        . ${SCRIPT_DIR}/deepops/proxy.sh && ansible-galaxy collection install --force -r roles/requirements.yml >/dev/null
-        . ${SCRIPT_DIR}/deepops/proxy.sh && ansible-galaxy role install --force -r roles/requirements.yml >/dev/null
-    else
-        ansible-galaxy collection install --force -r roles/requirements.yml >/dev/null
-        ansible-galaxy role install --force -r roles/requirements.yml >/dev/null
-    fi
-
-
+    as_user ansible-galaxy collection install --force -r roles/requirements.yml >/dev/null
+    as_user ansible-galaxy role install --force -r roles/requirements.yml >/dev/null
 else
-    echo "ERROR: Unable to install Ansible Galaxy roles"
+    echo "ERROR: Unable to install Ansible Galaxy roles, 'ansible-galaxy' command not found"
 fi
 
 # Update submodules
-git status >/dev/null 2>&1
-if [ $? -eq 0 ] ; then
-    if [ $PROXY_USE -gt 0 ]; then
-        . ${SCRIPT_DIR}/deepops/proxy.sh && git submodule update --init
-    else
-        git submodule update --init
-    fi
+if command -v git &> /dev/null ; then
+    as_user git submodule update --init
 else
-    echo "ERROR: Unable to update Git submodules"
+    echo "ERROR: Unable to update Git submodules, 'git' command not found"
 fi
 
 # Copy default configuration
-CONFIG_DIR=${CONFIG_DIR:-./config}
-if [ ! -d "${CONFIG_DIR}" ] ; then
-    cp -rfp ./config.example "${CONFIG_DIR}"
-    echo "Copied default configuration to ${CONFIG_DIR}"
-else
-    echo "Configuration directory '${CONFIG_DIR}' exists, not overwriting"
+if grep -i deepops README.md >/dev/null 2>&1 ; then
+    if [ ! -d "${CONFIG_DIR}" ] ; then
+        cp -rfp ./config.example "${CONFIG_DIR}"
+        echo "Copied default configuration to ${CONFIG_DIR}"
+    else
+        echo "Configuration directory '${CONFIG_DIR}' exists, not overwriting"
+    fi
 fi
+
+# Add Ansible virtual env to PATH when using Bash
+if [ -f "${VENV_DIR}/bin/activate" ] ; then
```

Comment:
I'm unsure whether I like doing this by default, or whether I'd prefer to print instructions for the user to do this.

If possible, a flag to enable/disable this would be handy.